.travis.yml

```diff
@@ -23,19 +22,18 @@ script:
   - ./vendor/bin/phpunit
 
 jobs:
-  allow_failures:
-    - php: 7.4snapshot
-
   include:
     - stage: Test
       env: DEPENDENCIES=low
+      php: 7.2
```

Comment:
Not needed since `7.2` is the lowest version already.

Reply:
Wasn't sure what version it picks, but if we can be sure it's 7.2 I'll drop it.